docs/src/pages/components/slider-styled/CustomizedSlider.js

```diff
@@ -1,19 +1,17 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Slider from '@material-ui/lab/SliderStyled';
-import styled from '@emotion/styled';
 import Typography from '@material-ui/core/Typography';
 import Tooltip from '@material-ui/core/Tooltip';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    width: `calc(300px + ${theme.spacing(6)})`,
-  },
-  margin: {
-    height: theme.spacing(3),
-  },
-}));
+const Root = styled('div')`
+  width: ${(props) => `calc(300px + ${props.theme.spacing(6)})`};
+`;
```

Comment:
What do you think about https://material-ui.com/guides/interoperability/#theme?

```suggestion
const Root = styled('div')`${({ theme }) => `
  width: calc(300px + ${theme.spacing(6)});
  `}
`;
```
or maybe simpler:

```suggestion
const Root = styled('div')(({ theme }) => `
  width: calc(300px + ${theme.spacing(6)});
  `
);

Reply:
I like the second one better 👌 will make that one convention (not sure if we use Box how much it will be used, but let’s see :)

Reply:
Done